packages/mui/src/FieldHelpTemplate/FieldHelpTemplate.tsx

```diff
@@ -10,14 +11,14 @@ export default function FieldHelpTemplate<
   S extends StrictRJSFSchema = RJSFSchema,
   F extends FormContextType = any,
 >(props: FieldHelpProps<T, S, F>) {
-  const { fieldPathId, help } = props;
+  const { fieldPathId, help, uiSchema, registry } = props;
   if (!help) {
     return null;
   }
-  const id = helpId(fieldPathId);
+
   return (
-    <FormHelperText component='div' id={id}>
-      {help}
+    <FormHelperText id={helpId(fieldPathId)} style={{ marginTop: '5px' }}>
```

Comment:
Please restore the `component='div'`
```suggestion
    <FormHelperText component='div' id={helpId(fieldPathId)} style={{ marginTop: '5px' }}>
```